doc/conf.py

```diff
@@ -77,6 +79,8 @@
 numpydoc_show_inherited_class_members = False
 numpydoc_class_members_toctree = False
 
+autodoc_mock_imports = ["dash", "holoviews.doc_convert", "js", "pytest", "pyodide"]
```

Comment:
For my own understanding, could you please briefly explain why this is needed?

Reply:
Some of the warnings related to "failed imports" are actually optional dependencies that only work in certain environments but the sphinx build still shows warnings regarding them. So by mocking the missing modules, these warnings are suppressed.